backend/src/test/java/com/daedan/festabook/place/controller/PlaceGeographyControllerTest.java

```diff
@@ -116,6 +117,27 @@ class getAllPlaceGeographyByOrganizationId {
                     .statusCode(HttpStatus.OK.value())
                     .body("$", hasSize(expectedSize));
         }
+
+        @Test
+        void 성공_Coordinate가_없을_경우_null() {
+            // given
+            Organization organization = OrganizationFixture.create();
+            organizationJpaRepository.save(organization);
+
+            Place place = PlaceFixture.create(organization, PlaceCategory.BAR, null);
+            placeJpaRepository.saveAll(List.of(place));
+
+            // when & then
+            RestAssured
+                    .given()
+                    .header("organization", organization.getId())
+                    .when()
+                    .get("/places/geographies")
+                    .then()
+                    .statusCode(HttpStatus.OK.value())
+                    .body("[0].markerCoordinate.latitude", equalTo(null))
+                    .body("[0].markerCoordinate.longitude", equalTo(null));
```

Comment:
```suggestion
                    .body("[0].markerCoordinate.latitude", nullValue())
                    .body("[0].markerCoordinate.longitude", nullValue());
```

`nullValue()`는 어떤가요?

Reply:
반영했어요!